Yoast/wordpress-seo
# Remove the table in the user profile page

## Summary

- removes the table from the "Yoast SEO settings" section in the user profile page
- introduces new CSS to mimic the WordPress two columns layout

## Relevant technical choices:

I've tried to make the new CSS generic enough to be reusable for other forms too.
This is just a first round, far from being perfect and complete. New rules and adjustments can be added in future iterations.

Worth reminding WordPress now uses "native fonts" so it's really not possible to have a pixel-perfect layout across different browsers and platforms, due to the different font types used and differences in the font rendering engine.

Tried to use BEM naming conventions.
**Needs decision: instead of `.yoast-settings` we could use a more generic `.yoast-form` naming convention.**

Tested trying to reproduce different use cases and edge cases: with and without fieldsets, with very long label text, etc. See screenshots below.

## Test instructions

This PR can be tested by following these steps:

- build the CSS
- check the layout f the "Yoast SEO settings" section in `/wp-admin/profile.php`
- check responsiveness shrinking your browser's window at different widths 
- check the responsive view under 782px 

Fixes #6542 

![screen shot 2017-01-31 at 16 43 52-fullpage](https://cloud.githubusercontent.com/assets/1682452/22474445/168d38fe-e7dc-11e6-831d-c5cc38f6cdc8.png)

![screen shot 2017-01-31 at 16 33 55-fullpage](https://cloud.githubusercontent.com/assets/1682452/22474444/16897656-e7dc-11e6-9428-a548e6fd7507.png)
